src/axolotl/utils/data/lock.py

```diff
@@ -46,18 +46,20 @@ def load(self, load_fn: Callable[[], Any]) -> Any:
     def _increment_counter(self):
         """Safely increment the process counter."""
         if self.counter_path.exists():
-            count = int(self.counter_path.read_text().strip())
+            counter_content = self.counter_path.read_text().strip()
+            count = int(counter_content) if counter_content else 0
```

Comment:
I don't know why `counter_content` would be something other than an int if it exists. If it exists, it should have an int written to it. Probably the file was mid-write when this was originally hit. Are you able to reproduce the original error?

But hey, if this works, we should go with it for now. We may need to be more cleve with the counter file handling.